README.md

```diff
@@ -14,6 +14,8 @@ SPDX-License-Identifier: MIT
 
 The goal of the project is to build an open-source platform that empowers researchers to manage simple studies without needing deep technical expertise. This repository contains the frontend code for the platform. Visit [spezi-web-service-study-platform](https://github.com/StanfordSpezi/spezi-web-service-study-platform) for the backend code.
 
+<img width="1618" height="939" alt="Spezi Web Study Platform interface overview" src="https://github.com/user-attachments/assets/a96f9151-3d1a-4eaa-bad9-cdf1ad61f9f1" />
```

Comment:
Would be good to bundle the images right in the repo under version control. Would also be good to add a dark and light mode version similar to the other Spezi repos 👍